{
    "type": "bundle",
    "id": "bundle--5a5b817a-09b9-47e7-a9af-ad3a8fef3213",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--3e822fd5-3ca4-4841-b31a-fbdb0c8d2b63",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-10-22T01:45:31.683575Z",
            "modified": "2024-10-22T01:45:31.683575Z",
            "name": "CVE-2022-48990",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\ndrm/amdgpu: fix use-after-free during gpu recovery\n\n[Why]\n    [  754.862560] refcount_t: underflow; use-after-free.\n    [  754.862898] Call Trace:\n    [  754.862903]  <TASK>\n    [  754.862913]  amdgpu_job_free_cb+0xc2/0xe1 [amdgpu]\n    [  754.863543]  drm_sched_main.cold+0x34/0x39 [amd_sched]\n\n[How]\n    The fw_fence may be not init, check whether dma_fence_init\n    is performed before job free",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-48990"
                }
            ]
        }
    ]
}